File: src/mappings/file_browser.rs
pub fn get(filetype: &str) -> Option<(&str, &str)> {
    let file_browser = match filetype {
        "netrw" => ("default", "Netrw"),
        "TelescopePrompt" => ("telescope", "Telescope"),
        "dirvish" => ("default", "Dirvish"),
        "fern" => ("default", "Fern"),
        "oil" => ("default", "Oil"),
        "neo-tree" => ("default", "Neo-Tree"),
        "NvimTree" => ("default", "nvim-tree"),
        "minifiles" => ("default", "mini.files"),
        "yazi" => ("default", "Yazi"),
        _ => return None,
    };

    Some(file_browser)
}

File: lua/cord/mappings/file_browser.lua
local M = {}

local mappings = {
  netrw = { 'default', 'Netrw' },
  TelescopePrompt = { 'telescope', 'Telescope' },
  dirvish = { 'default', 'Dirvish' },
  fern = { 'default', 'Fern' },
  oil = { 'default', 'Oil' },
  NvimTree = { 'default', 'nvim-tree' },
  minifiles = { 'default', 'mini.files' },
  yazi = { 'default', 'Yazi' },
  ['neo-tree'] = { 'default', 'Neo-Tree' },
}

M.get = function(filetype) return mappings[filetype] end

return M
